snapcraft/__init__.py

```diff
@@ -138,15 +138,15 @@
 
   - build-packages: [deb, deb, deb...]
 
-    A list of Ubuntu packages to install on the build host before building
+    A list of packages to install on the build host before building
```

Comment:
It would be worth explaining where are the packages from. For now, maybe add: currently, only packages from the ubuntu archive of the currently installed distribution are supported.